src/pfe/file-watcher/server/src/controllers/projectEventsController.ts

```diff
@@ -78,6 +78,8 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
         let isSettingFileChanged = false;
         let isProjectBuildRequired = false;
 
+        const projectHandler = await projectExtensions.getProjectHandler(projectInfo);
```

Comment:
Since you got the project handler here, you may not need to get it again later inside this function (I think there are 2 places below that calls `await projectExtensions.getProjectHandler`

Reply:
updated